src/core/client/stream/components/Stream.tsx

```diff
@@ -1,18 +1,55 @@
+import { Localized } from "fluent-react/compat";
 import * as React from "react";
 import { StatelessComponent } from "react";
 
+import { Button } from "talk-ui/components";
+
 import CommentContainer from "../containers/CommentContainer";
+import PostCommentFormContainer from "../containers/PostCommentFormContainer";
+import ReplyListContainer from "../containers/ReplyListContainer";
+import Logo from "./Logo";
 
 export interface StreamProps {
-  comments: ReadonlyArray<{ id: string }>;
+  assetID: string;
+  isClosed: boolean;
+  comments: ReadonlyArray<{ id: string }> | null;
+  onLoadMore: () => void;
+  hasMore: boolean;
+  disableLoadMore: boolean;
 }
 
 const Stream: StatelessComponent<StreamProps> = props => {
+  if (props.comments === null) {
+    // TODO: (@cvle) What's the reason for this being null?
```

Comment:
@wyattjoh Maybe you can give an answer here :-)

Reply:
No reason it seems! Feel free to mark all the `CommentsConnection` returns as `!`.